lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -134,16 +143,22 @@ function dosomething_campaign_preprocess_common_vars(&$vars, &$wrapper) {
         ),
       ),
       'twitter' => array(
```

Comment:
@chloealee we wouldn't want this tied specifically to organ-donation checkbox, we would want it tied to the checkbox you created in this PR